samples/Chat/src/app/App.tsx

```diff
@@ -16,7 +15,6 @@ console.info(`Thread chat sample using @azure/communication-chat : ${getChatSDKV
 console.info(`Build Date : ${getBuildTime()}`);
 
 initializeIcons();
-registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
```

Comment:
Why is it not needed anymore?

Reply:
Is it because, composites registerIcons be default?

Reply:
Yeah the base composite registers them anyway and then takes in the icons its overriding from the prop. this causes the overriden icons to be hidden if left in.